src/index.js

```diff
@@ -35,17 +31,19 @@ export default class SplunkEvents {
     });
   }
 
-  logEvent(event) {
+  logEvent(host, level, type, workflowType, workflowInstance, event) {
```

Comment:
I think that host should be configured by the user because this is a parameter that probably will not change but level, type, workflowtype and workflowInstance must be defined explicitly for each event to make the developer think about the criticality of each log.